VideoLocker/AndroidManifest.xml

```diff
@@ -93,28 +93,27 @@
 
         <activity
             android:name=".view.CourseDiscussionTopicsActivity"
+            android:configChanges="orientation|keyboardHidden|screenSize"
```

Comment:
Had an offline discussion with @1zaman over the reinstantiation approach, and we concluded that it would require more refactoring in places that we were initially expecting, adding the possibility of breaking things without us knowing i.e. will require thorough investigation into the code+testing.

So, for now we'll be using the config changes.